packages/web/src/components/play-bar/desktop/PlayBar.js

```diff
@@ -503,22 +508,40 @@ class PlayBar extends Component {
 const makeMapStateToProps = () => {
   const getCurrentQueueItem = makeGetCurrent()
 
-  const mapStateToProps = (state, props) => ({
-    accountUser: getAccountUser(state),
-    currentQueueItem: getCurrentQueueItem(state),
-    playCounter: getCounter(state),
-    collectible: getCollectible(state),
-    isPlaying: getPlaying(state),
-    isBuffering: getBuffering(state),
-    playingUid: getPlayingUid(state),
-    playbackRate: getPlaybackRate(state),
-    lineupHasTracks: getLineupHasTracks(
+  const mapStateToProps = (state) => {
+    const premiumTrackSignatureMap = getPremiumTrackSignatureMap(state)
+    const lineupEntries = getLineupEntries(
       getLineupSelectorForRoute(state),
       state
-    ),
-    userId: getUserId(state),
-    theme: getTheme(state)
-  })
+    )
+    const lineupHasAccessibleTracks = lineupEntries.some((entry) => {
+      if (entry.kind !== Kind.TRACKS || !entry.id) return true
```

Comment:
yeah... glad you brought this up. i was a bit hesitant on what the logic here could be. updated it.
- no need for !entry.id check because id should just be there
- returning false if kind is not 'TRACKS'
thanks